### PR TITLE
feat(Core/Commands): Arena command

### PR DESCRIPTION
**Pull request by IntelligentQuantum**
##### TESTS PERFORMED:
Yes, builded without errors. Tested in game, all commands work without problem.

##### HOW TO TEST THE CHANGES:
Try these commands :

1: .arena create $name "arena name" #type (2/3/5)
2: .arena disband #TeamID
3: .arena rename "oldname" "newname"
4: .arena captain #TeamID $NewCapitanName
5: .arena info #TeamID
6: .arena lookup $name

##### KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

##### Target branch(es):

Master

### CODE REVIEW

**Comment by poszer** [comment]
Could you please fix travis errors? @IntelligentQuantum

File: data/sql/updates/pending_db_world/rev_1557081152867649198.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1557081152867649198');
+
+INSERT INTO `command` (`name`, `security`, `help`) VALUES ('arena create', 3, 'Syntax: .arena create $name "arena name" #type\n\nA command to create a new Arena-team in game. #type  = [2/3/5]');
+INSERT INTO `command` (`name`, `security`, `help`) VALUES ('arena disband', 3, 'Syntax: .arena disband #TeamID\n\nA command to disband Arena-team in game.');
+INSERT INTO `command` (`name`, `security`, `help`) VALUES ('arena rename', 3, 'Syntax: .arena rename "oldname" "newname"\n\nA command to rename Arena-team name.');
+INSERT INTO `command` (`name`, `security`, `help`) VALUES ('arena captain', 3, 'Syntax: .arena captain #TeamID $name\n\nA command to set new captain to the team $name must be in the team');
+INSERT INTO `command` (`name`, `security`, `help`) VALUES ('arena info', 2, 'Syntax: .arena info #TeamID\n\nA command that show info about arena team');
+INSERT INTO `command` (`name`, `security`, `help`) VALUES ('arena lookup', 2, 'Syntax: .arena lookup $name\n\nA command that give a list of arenateam with the given $name');
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1557081152867649198.sql]: thanks for your contribution! Just a few notes:

- there is no need to repeat `INSERT INTO `command` (`name`, `security`, `help`) VALUES` more than once
- always use `DELETE` before `INSERT`

Please have a look at: http://www.azerothcore.org/wiki/Dealing-with-SQL-files

File: src/server/scripts/Commands/cs_arena.cpp [hunk 2]
```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>, released under GNU GPL v2 license: https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-GPL2
+ * Copyright (C) 2008-2016 TrinityCore <http://www.trinitycore.org/>
+ * Copyright (C) 2005-2009 MaNGOS <http://getmangos.com/>
+ */
```
Comment (FrancescoBorzi) [inline src/server/scripts/Commands/cs_arena.cpp]: has this file been imported from TC?

File: data/sql/updates/pending_db_world/rev_1557081152867649198.sql (hunk 1 above)
Comment (poszer) [reply data/sql/updates/pending_db_world/rev_1557081152867649198.sql]: > thanks for your contribution! Just a few notes:
> 
> * there is no need to repeat `INSERT INTO `command` (`name`, `security`, `help`) VALUES` more than once
> * always use `DELETE` before `INSERT`
> 
> Please have a look at: http://www.azerothcore.org/wiki/Dealing-with-SQL-files

I corrected that for him

File: src/server/scripts/Commands/cs_arena.cpp (hunk 2 above)
Comment (IntelligentQuantum) [reply src/server/scripts/Commands/cs_arena.cpp]: Yes

File: data/sql/updates/pending_db_world/rev_1557081152867649198.sql (hunk 1 above)
Comment (IntelligentQuantum) [reply data/sql/updates/pending_db_world/rev_1557081152867649198.sql]: Tnx @poszer 

@FrancescoBorzi sorry for this mistake

**Comment by poszer** [comment]
Tested on Win OS:
Status : ========== Build: 7 succeeded, 0 failed, 7 up-to-date, 2 skipped ==========

Tested in game / Table below:

| Command | Working|
| --- | --- |
| .arena create (2/3/5) | Creating 2/3/5 team - Work |
| .arena disband | Command disband team - Work |
| .arena rename | Command rename teams - Work |
| .arena captain | Promoting other team captain - Work |
| .arena info | Giving info about arena team - Work |
| .arena lookup | Searching for teams - Work |


Every command provided in this PR works perfectly. There isn't any issue or conflict.
Thank you  @IntelligentQuantum for this :+1: 

Ready for merge.

**Pull request merged**